src/dev/src/app/tree-view/eager-declarative-tree/eager-declarative-tree.html

```diff
@@ -54,6 +54,44 @@ <h4>Selection</h4>
   </clr-tree-node>
 </div>
 
+<h4>Selection with Disabled nodes</h4>
+<div class="clr-example example-disabled-nodes">
+  <span>Click on the name of a node to select for toggling it's disabled state:</span>
+  <div>
+    <button class="btn" [disabled]="!selectedNodeForDisabling" [class.btn-success-outline]="isSelectedNodeDisabled"
+      [class.btn-danger-outline]="!isSelectedNodeDisabled" (click)="toggleDisableStateOfSelectedNode()">
+      {{ isSelectedNodeDisabled ? "Enable" : "Disable" }} node {{ selectedNodeForDisablingName }}
+    </button>
+  </div>
+  <clr-tree-node [clrSelected]="false" (click)="selectNodeForDisabling($event)">
```

Comment:
I'd much rather this demo be simpler, where you have a button to set the disabled state on a particular node (or several buttons to show different nodes) rather than the specialized logic and display changes to make the titles clickable. I don't want our demos to override the style or behavior of a component just to demonstrate a different feature, as people sometimes see these dev examples and think its a feature. Also it would make it easier to have a demo for each of the permutations.